cms/static/js/views/assets.js

```diff
@@ -308,29 +333,32 @@ define(["jquery", "underscore", "gettext", "js/views/baseview", "js/models/asset
             },
 
             displayFinishedUpload: function (resp) {
-                var asset = resp.asset;
+                var asset = resp.asset,
+                    $progressFill = $('.upload-modal .progress-fill');
 
                 $('.upload-modal h1').text(gettext('Upload New File'));
                 $('.upload-modal .embeddable-xml-input').val(asset.portable_url).show();
                 $('.upload-modal .embeddable').show();
                 $('.upload-modal .file-name').hide();
-                $('.upload-modal .progress-fill').html(resp.msg);
+                HtmlUtils.setHtml($progressFill, resp.msg);
```

Comment:
Do you know if ``resp.msg`` includes HTML? If it does, you should use:

    HtmlUtils.setHtml($progressFill, HtmlUtils.HTML(resp.msg));

If it doesn't, then you can just use this:

    $progressFill.text(resp.msg);